### PR TITLE
Add eid belgium

### PR DESCRIPTION
**Pull request by wget**
## Description
Make the package work again. Rewrite it and make it automatic.

## Motivation and Context
Have auto update to this package.

## How Has this Been Tested?
Windows 10 Pro with DriverStore reset each time with the excellent RAPR (DriverStore Explorer).
Install and uninstall process are both working.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

## Original Location
- [Original Repository](https://github.com/Vanclec/chocolatey-eid-belgium)
- Open Issues: N/A
- [x] *Include the link to the opened PR that removes the package from the original location* -> [deprecation link](https://github.com/Vanclec/chocolatey-eid-belgium/pull/1)
- [x] The [migration guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Package-migration-process) have been followed

### CODE REVIEW

File: automatic/eid-belgium/tools/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```
Comment (AdmiringWorm) [inline automatic/eid-belgium/tools/LICENSE.txt]: please move this file to a `legal` directory instead, to match the convention used for most of the existing packages.

You'll also need to update the files section of course.

File: automatic/eid-belgium/tools/VERIFICATION.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+VERIFICATION
```
Comment (AdmiringWorm) [inline automatic/eid-belgium/tools/VERIFICATION.txt]: please move this file to a `legal` directory instead, to match the convention used for most of the existing packages.

File: automatic/eid-belgium/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName    = 'eid-belgium'
+  fileType       = 'msi'
+  softwareName   = 'eid-belgium*'
```
Comment (AdmiringWorm) [inline automatic/eid-belgium/tools/chocolateyinstall.ps1]: this doesn't seem to be correct, is this actually what is shown in `Add/Remove programs` in the control panel?

**Comment by AdmiringWorm** [comment]
This is notification that this PR will be closed in a short while unless the requested changes have been fixed and/or responded to.

File: automatic/eid-belgium/tools/LICENSE.txt (hunk 1 above)
Comment (wget) [reply automatic/eid-belgium/tools/LICENSE.txt]: Taken care of in https://github.com/wget/chocolatey-package-eid-belgium

File: automatic/eid-belgium/tools/VERIFICATION.txt (hunk 2 above)
Comment (wget) [reply automatic/eid-belgium/tools/VERIFICATION.txt]: Taken care off in https://github.com/wget/chocolatey-package-eid-belgium

File: automatic/eid-belgium/tools/chocolateyinstall.ps1 (hunk 3 above)
Comment (wget) [reply automatic/eid-belgium/tools/chocolateyinstall.ps1]: You were right. This was `Belgium e-ID` with a long suffix. Taken care off in https://github.com/wget/chocolatey-package-eid-belgium

**Comment by wget** [comment]
Like requested this package has now been ported in my own repo:  https://github.com/wget/chocolatey-package-eid-belgium

**Pull request closed without merging**